### PR TITLE
Remove MFA Codes / Requests

### PR DESCRIPTION
**Pull request by sampoder**
RIP

### CODE REVIEW

File: db/schema.rb
Comment (garyhtou) [inline db/schema.rb:1]: Can you remove the unrelated changes?

**Review by garyhtou — CHANGES_REQUESTED** [review]
this looks good. it's just the unrelated changes

**Comment by sampoder** [comment]
sorry i missed your comment earlier @garyhtou, should be done now!

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -2,22 +2,9 @@
 
 class AdminController < ApplicationController
   skip_after_action :verify_authorized # do not force pundit
-  skip_before_action :signed_in_user, only: [:twilio_messaging]
-  skip_before_action :verify_authenticity_token, only: [:twilio_messaging] # do not use CSRF token checking for API routes
-  before_action :signed_in_admin, except: [:twilio_messaging]
```
Comment (maxwofford) [inline app/controllers/admin_controller.rb]: I might be misreading this. Do we want to do this?

Comment (sampoder) [reply app/controllers/admin_controller.rb]: Mb! We don't want to remove:

```
before_action :signed_in_admin, except: [:twilio_messaging]
```

Comment (sampoder) [reply app/controllers/admin_controller.rb]: The other two we want to remove though to as they were only for `:twilio_messaging`.

This should be fixed now!

**Review by cjdenio — APPROVED** [review]
This looks ready to merge!

**Pull request merged**